bpf/process/types/basic.h

```diff
@@ -1280,6 +1280,78 @@ filter_64ty_map(struct selector_arg_filter *filter, char *args)
 	return 0;
 }
 
+#ifdef __LARGE_BPF_PROG
+FUNC_LOCAL long
+filter_64ty_range(struct selector_arg_filter *filter, char *args)
+{
+	__u64 *v = (__u64 *)&filter->value;
+	int all = 0, i, j = 0;
+
+#pragma unroll
+	for (i = 0; i < MAX_MATCH_VALUES; i++) {
+		__u64 start = v[i * 2], end = v[i * 2 + 1];
+		bool res;
+
+		if (is_signed_type(filter->type))
+			res = ((__s64)start <= *(s64 *)args) && (*(s64 *)args <= (__s64)end);
```

Comment:
since your range is inclusive on both ends, how would that work with:
```
    values:
    - "0:187904818"
    - "187904818:1879048187"
 ```
technically both range would match for number `187904818`. I imagine it's not a big deal since you short-circuit as soon as you match. Maybe you want Go-like range with `slice[0:5]` being `0, 1, 2, 3, 4` elements. Anyway that would be great eventually to have it documented somewhere how they behave.

Reply:
the [min,max] range seems more user friendly for me.. I added docs